### DISCUSSION TITLE
Docstring correction for ge function

### DISCUSSION THREAD

**Discussion by sandeep026** [casadi/casadi#3463] [Q&A] [answered]
Less than or equal to must be replaced by greater than or equal.
```
    """
      Logical greater or equal to: (x,y) -> x <= y.

    ::

      ge(float x, float y) -> float
      ge(DM x, DM y) -> DM
      ge(SX x, SX y) -> SX
      ge(MX x, MX y) -> MX


    Extra doc: https://github.com/casadi/casadi/wiki/L_ou

    Doc source: 
    https://github.com/casadi/casadi/blob/develop/casadi/core/generic_expression.hpp#L176

    Implementation: 
    https://github.com/casadi/casadi/blob/develop/casadi/core/generic_expression.hpp#L176-L178




    """
```

```
Logical greater or equal to: (x,y) -> x >= y.
```

**Comment by jgillis** [comment, marked as answer]
Thank you for reporting, locally fixed now!

Since you are scrutinizing our documentation, do feel welcome to make pull requests (use that implementation or doc source link to find the appropriate file to modify), or otherwise add things in the wiki (use that extra doc link)

**Reply by sandeep026** [reply]
I would love to. I haven't made any contributions yet and use git locally for version control. Will learn more about pull request and guidelines.

How does the wiki work and is it reviewed? I edited the documentation page and added a message. Will that also reflect in the source code?

"Extra documentation" heading has writing tips hyperlink but redirects to itself. What is the purpose?

**Reply by jgillis** [reply]
I've added some writing tips now. Note that the `<=` to `>=` edit was already added to the source by me https://github.com/casadi/casadi/commit/a66c7b28d56a745aaa5c9b7e128139f9c1b99d4a

**Reply by sandeep026** [reply]
ok
